interceptor/translator.go

```diff
@@ -29,7 +37,7 @@ func (n *translatorImpl) TranslateResponse(resp any) (bool, error) {
 	return n.visitor(resp, n.matchResp)
 }
 
-func createNameTranslator(mapping map[string]string) matcher {
+func createNameMatcher(mapping map[string]string) matcher {
```

Comment:
just to be more explicit
name `matcher` as `stringMatcher` and `createNameMatcher` as `createStringMatcher` ?